common/models/user.js

```diff
@@ -190,18 +133,17 @@ module.exports = (User) => {
   User.remoteMethod('getAllUser', getAllUser)
   
 
-  //7th API for Detete User
+  //6th API for Detete User
   User.deleteUser = (id, cb) => {
     let query = { _id:id }
     let options = { status:"DELETED" }
-    User.update(query , options, (err, user) => {
-      if (err)
-        cb(err)
-      else if(user.count == 0)
+    User.update(query, options).then(user=> {
```

Comment:
why you use update query, instead of this use findOneAndRemove

Reply:
When i use findOneAndUpdate or findByIdAndUpdate instead of update it show me error ......findByIdAndUpdate or findOneAndUpdate is not function.